wildcard/media/convert.py

```diff
@@ -203,9 +204,9 @@ def _convertFormat(context):
                 logger.warning('error converting to %s' % video_type)
                 continue
             if os.path.exists(output_filepath):
-                fi = open(output_filepath)
-                namedblob = NamedBlobFile(
-                    fi, filename=switchFileExt(video.filename,  video_type))
+                with open(output_filepath, 'rb') as fi:
+                    namedblob = NamedBlobFile(
+                        fi, filename=switchFileExt(video.filename,  video_type))
                 setattr(context, fieldname, namedblob)
                 fi.close()
```

Comment:
`fi.close` should not be necessary because `fi` should be closed by the context manager (which is a really nice addition, BTW).

Reply:
I have removed the unnecessary lines. Thanks for reply.

Reply:
Thanks for the fixes!